flight/PiOS/STM32/pios_servo.c

```diff
@@ -704,7 +715,11 @@ static int DSHOT_Update()
 		PIOS_Assert(info->value < 2048);
 
 		uint16_t message = info->value << 5;
-		/* Don't set telem req bit for now */
+
+		/* Set telem bit on commands */
+		if ((info->value > 0) && (info->value < 48)) {
```

Comment:
This is a stupid, recent change in blheli.  It doesn't mean to actually send telem in this case-- they send it out at the same time to all channels.  It's uh, another "check bit" on commands, kind of?

Reply:
Wut.